backend/molgenis-emx2-rdf/src/test/resources/rdf_files/rdf_api/pet_store/emx2/schema.ttl

```diff
@@ -427,6 +427,19 @@ PetStoreNr1:Tag a owl:Class;
   <http://localhost:8080/petStoreNr1/api/rdf/Pet/column/mg_insertedOn> "2025-06-16T15:01:15"^^xsd:dateTime;
   <http://localhost:8080/petStoreNr1/api/rdf/Pet/column/mg_updatedOn> "2025-06-16T15:01:15"^^xsd:dateTime .
 
+<http://localhost:8080/petStoreNr1/api/rdf/Pet/name=yakul>
```

Comment:
I think we should prevent these kinds of tests. Tests that use pet store ideally do specific verifications such that additions to that schema don't have impact.

N.B. when in backend we wouldn't change the pet store schema but instead create a temp schema and mutate that one. Might you not better served by having a 'draft' in another demo schema?

Primary purpose of pet store s for integration tests in backend.

Having said that, if this is the only test that fails than it is good that we found this breaking test because as per my first point, tests on pet store ideally don't break if 'anything' changes by using more specific testing assertions.